Server.js

```diff
@@ -77,6 +80,93 @@ app.get('*', function(req, res){
   res.status(404).send('Looks like you have lost the path ;)');
 })
 
+app.use(express.json())
+
+app.post('/api/task', function(req, res){
+  var data = fs.readFileSync("TasksFile.JSON")
+  var JSON_obj_file = JSON.parse(data)
+  
+  var JSON_obj_task = req.body
+
+  JSON_obj_task.createTime = GetFullTime()
+
+  //JSON_obj_task.id = new_task_id
+
+  JSON_obj_file[new_task_id] = JSON_obj_task
+  new_task_id += 1
```

Comment:
use UUID, what happen if I close the server and open it again? I could have two tasks with the same id